tecs/tecs.cpp

```diff
@@ -208,8 +208,8 @@ void TECS::_update_speed_setpoint()
 
 	// Calculate limits for the demanded rate of change of speed based on physical performance limits
 	// with a 50% margin to allow the total energy controller to correct for errors.
-	float velRateMax = 0.5f * _STE_rate_max / _tas_state;
-	float velRateMin = 0.5f * _STE_rate_min / _tas_state;
+        float velRateMax = 0.5f * _STE_rate_max / _tas_state;
```

Comment:
Could you revert these whitespace changes?

Reply:
Done, thanks for pointing that out.